rotkehlchen/chain/ethereum/utils.py

```diff
@@ -147,8 +147,8 @@ def multicall_2(
     Use a MULTICALL_2 contract for an aggregated query. If require_success
     is set to False any call in the list of calls is allowed to fail.
     """
-    return ETH_MULTICALL_2.call(
-        manager=ethereum,
+    return ETH_MULTICALL_2[ChainID.ETHEREUM].call(
```

Comment:
Eventually this should search for the multicall bases on the ChainID of the manager?

Also thinking about it shouldn't this be a method of the manager itself? That first `manager` argument looks a lot like a `self`. This way there would be no need for a check for the proper multicall addresses as it would be initialized with them

Reply:
That actually makes a lot of sense!

> Eventually this should search for the multicall bases on the ChainID of the manager?

This is what I did for ETH_SCAN and I'm replicating the same for ETH_MULTICALL.